.gitignore

```diff
@@ -5,6 +5,8 @@ venv
 roles/galaxyproject.galaxy/
 roles/galaxyproject.nginx/
 roles/galaxyproject.postgresql/
+# roles/galaxyproject.slurm/ # Commented out as we have made modifications to this role so slurmdbd installation works
```

Comment:
Could the modifications be flagged in the slurm role so that we can see where it is different from galaxyproject.slurm?  Something like `# modified for usegalaxy-au/infrastructure`

Reply:
There are a lot of changes. I can put in comments.